Standards/scs-0302-v1-domain-manager-role.md

```diff
@@ -55,7 +55,7 @@ To address this, this standard defines a new Domain Manager role in conjunction
 3. The cloud admin creates one or more users within each of the applicable domains and assigns the Domain Manager role to them. These users represent the Domain Managers of the corresponding domain.
 4. The customer uses the Domain Manager users to manage (create, update, delete) users, projects, groups and corresponding role assignments within their domain.
 
-## Design Considerations
+## Context
```

Comment:
This is a standard. Why the change?

Reply:
I thought I removed the commit containing this the changes to the Standards and DRs, since we only recommend the section names now. I will revert these changes then in another commit.

Reply:
Updated in 37efde65c0bd6794d7442454bc77785ed92a3a06